daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1347,35 +1347,12 @@ Safe strncpy that ensures a trailing zero
 =============
 */
 
-#ifndef NDEBUG
```

Comment:
I thought it was another chunk. I thought that noone uses that feature of Q_strncpyz so I might as well remove it but I'm not attached to that.